process/factory/metachain/vmContainerFactory_test.go

```diff
@@ -107,3 +110,41 @@ func TestVmContainerFactory_Create(t *testing.T) {
 	acc := vmf.BlockChainHookImpl()
 	assert.NotNil(t, acc)
 }
+
+func makeGasSchedule() map[string]map[string]uint64 {
+	gasSchedule := make(map[string]map[string]uint64)
+	FillGasMapInternal(gasSchedule, 1)
+	return gasSchedule
+}
+
+func FillGasMapInternal(gasMap map[string]map[string]uint64, value uint64) map[string]map[string]uint64 {
+	gasMap["BaseOperationCost"] = FillGasMapBaseOperationCosts(value)
```

Comment:
Maybe we should declare some consts for these. In this way we can avoid typos that will lead to errors in tests.

Reply:
done